metaflow/plugins/kfp/kfp.py

```diff
@@ -623,6 +640,13 @@ def build_kfp_dag(
 
                 visited[node.name] = container_op
 
+                pytorch_deco = kfp_component.pytorch_decorator
+                if pytorch_deco:
+                    # TODO: Delete the volume.  This is a Q1 work item.
```

Comment:
Do we have a ticket for this? If we don't delete the volume what happens, are we on the hook for the cost of it?

Reply:
Is the solution just to use RPC instead?